core/src/main/java/google/registry/tools/UniformRapidSuspensionCommand.java

```diff
@@ -138,6 +148,21 @@ protected void initMutatingEppToolCommand() {
     } else {
       statusesToApply = URS_LOCKS;
     }
+
+    if (renewOneYear) {
+      setSoyTemplate(DomainRenewSoyInfo.getInstance(), DomainRenewSoyInfo.RENEWDOMAIN);
+      DomainBase domainBase = domain.get();
```

Comment:
There's also several calls to `domain.get()` above that you haven't harmonized.  If you want a local variable for this, then use it everywhere.  I'd change line 127 to:

```
    Optional<DomainBase> domainOpt = loadByForeignKey(DomainBase.class, domainName, now);
    checkArgumentPresent(domainOpt, "Domain '%s' does not exist or is deleted", domainName);
    DomainBase domain = domainOpt.get();
```

And then just use `domain` everywhere thereafter.